### PR TITLE
Backport to 3.4 of table format for alg paramters - GDAL raster misc

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429. Backport of #4474

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst [hunk 1]
```diff
@@ -23,114 +23,87 @@ This algorithm is derived from the `GDAL addo utility <https://www.gdal.org/gdal
...
 
 Outputs
 .......
 
-``Output layer`` [raster]
-  Output raster layer with overviews.
-
-.. _gdalbuildvirtualraster:
-
-Build virtual raster
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst]: Disappeared from the PR...

Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst]: For some reason, I have marked this as a 3.6 feature...  But it is definitely available in 3.4! I will check a bit, and add it back.

File: source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst [hunk 2]
```diff
@@ -147,62 +255,112 @@ This algorithm is derived from the `GDAL merge utility <https://www.gdal.org/gda
...
+       * 3 --- UInt32
+       * 4 --- Int32
+       * 5 --- Float32
+       * 6 --- Float64
+       * 7 --- CInt16
+       * 8 --- CInt32
+       * 9 --- CFloat32
+       * 10 --- CFloat64
+       
+   * - **Input pixel value to treat as "nodata**
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst]:
```suggestion
   * - **Input pixel value to treat as "nodata"**
```
maybe?

File: source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst]: Done.

File: source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst (hunk 2 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/rastermiscellaneous.rst]: Should be fixed for both 3.4 and master now. Thanks!

**Pull request merged**